zerver/tests/test_narrow.py

```diff
@@ -368,17 +426,21 @@ def test_successful_get_old_messages(self):
         narrow = [['pm-with', 'othello@zulip.com']]
         self.get_and_check_messages(dict(narrow=ujson.dumps(narrow)))
 
-        narrow = [dict(operator='pm-with', operand='othello@zulip.com')]
+        narrow = [dict(operator='pm-with', operand='othello@zulip.com')] # type: ignore
```

Comment:
The issue here is that there is another variable called `narrow` which interferes with this. Currently, mypy lacks the ability to redefine variables. So the solution would be to change the name of the variable like this:
```
narrow_2 = [dict(operator='pm-with', operand='othello@zulip.com')]  # type: List[Dict[str, str]]
```
or you can just pass the data directly to the function:
```
self.get_and_check_messages(dict(narrow=ujson.dumps([dict(operator='pm-with', operand='othello@zulip.com')]))) 
```